lib/elastic_whenever/task/rule.rb

```diff
@@ -51,6 +56,10 @@ def self.rule_name(identifier, expression, commands)
         "#{identifier}_#{Digest::SHA1.hexdigest([expression, commands.map { |command| command.join("-") }.join("-")].join("-"))}"
       end
 
+      def self.rule_description(identifier, expression, commands)
+        "#{identifier} - #{expression} - #{commands.map { |command| command.join(" ") }.join(" - ")}"
```

Comment:
I guess that it is necessary to cut so that it does not exceed 512 characters.
https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutRule.html#API_PutRule_RequestSyntax

Reply:
Good shout, I will get that fixed.